clipper_admin/clipper_admin/docker/docker_metric_utils.py

```diff
@@ -4,6 +4,8 @@
 import os
 from ..version import __version__
 
+PROM_VERSION = "master"
```

Comment:
Let's use the latest release (`v2.1.0`) instead of master

Reply:
Haha that was my in first commit but I changed it, I'll update it